script/DeployBridge.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19 <=0.9.0;
+
+import { MiniMeBase } from "@vacp2p/minime/contracts/MiniMeBase.sol";
+import { BaseScript } from "./Base.s.sol";
+import { DeploymentConfig } from "./DeploymentConfig.s.sol";
+import { OptimismMintableMiniMeToken } from "../contracts/optimism/OptimismMintableMiniMeToken.sol";
+import { SNTPlaceHolder } from "../contracts/SNTPlaceHolder.sol";
```

Comment:
@3esmit How do you feel about renaming this one? I find "placeholder" extremely misleading. It's effectively a token controller, so how about we rename it to `SNTTokenController` or something like that?

Reply:
Indeed. SNTOptimismController.sol would be better

Reply:
Okay cool, will do in a follow-up PR

Reply:
Here's a PR that renames the contract accordingly https://github.com/logos-co/optimism-bridge-snt/pull/19